docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
Ah @joshbruce I figured it out:

In `package.json` we have a `prepublish` script that executes certain commands after `npm install` (which doesn't make much sense). I guess those have to be run before `website:serve`.

I'll keep this in mind when working on improving the dev workflow this week.